capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -30,17 +30,27 @@
 
   @Field(name="eventLongitude")
   double eventLongitude;
+
+  @Field(name="foodAvaliable")
+  boolean foodAvaliable;
+
+  @Field(name="requiredFee")
+  boolean requiredFee;
   
   @Reference
   List<Review> reviews;
+ 
 
-  public Event(Organization organization, String eventTitle,String eventDateTime, String eventDescription, double eventLatitude, double eventLongitude) {
+  public Event(Organization organization, String eventTitle,String eventDateTime, String eventDescription, double eventLatitude, double eventLongitude, boolean foodAvaliable, boolean requiredFee) {
```

Comment:
I'd replace `boolean` with `Boolean`, because then old test data that did not have these fields can still be used.

But then you've got to check for `null` value and default to false instead; something like this:

```
this.foodAvaliable = foodAvaliable == null ? false : foodAvaliable;
this.requiredFee = requiredFee == null ? false : requiredFee;
```